### ISSUE TITLE
TypeError: require(...).env is not a function

### ISSUE THREAD

**Issue by joetidee** [rstacruz/mocha-jsdom#27]
Am getting this error:

```
TypeError: require(...).env is not a function
      at Context.<anonymous> (node_modules/mocha-jsdom/index.js:52:22)
```

I have `mocha-jsdom` v1.1.0 and `jsdom` v11.0.0 installed am am using it like this:

```
require('babel-register');
var jsdom = require('mocha-jsdom');
const expect = require('chai').expect;

describe('insertFxRow()', () => {
        jsdom();

        it('has document', function () {
            var div = document.createElement('div')
            expect(div.nodeName).eql('DIV')
        })
    });
```

Any thing to do with [this](https://github.com/tmpvar/jsdom/issues/1820)?

**Comment by amalahussein** [comment]
same issue here..

**Comment by kcbroomall** [comment]
Looks like jsdom api was updated, but the old api is still supported.

Change line 53 in the module to:
```
    require('jsdom/lib/old-api').env(
```

**Comment by cgarnier** [comment]
This fix is not released yet, right ?

**Comment by cgarnier** [comment]
You should add the jsdom to use directly in the dependencies. Peer dependencies sux.

**Comment by geminiyellow** [comment]
so, @rstacruz , how do you think the PR #29 ?

**Comment by ben-crowhurst** [comment]
Has a new version been pushed to NPM? I'm still seeing this issue.

**Comment by beoss** [comment]
To fix until an update is published is to change your JSDOM requirement to: "jsdom": "7.2.2"

**Comment by glesperance** [comment]
Can we get a fix pushed? I need the latest version of JSDOM for my code to work

**Comment by rstacruz** [comment]
Fixed in v1.2.0 and v2.0.0!
